manim/utils/file_ops.py

```diff
@@ -23,6 +25,12 @@ def add_extension_if_not_present(file_name, extension):
         return file_name
 
 
+def add_version_before_extension(file_name):
+    index = str(file_name).index(Path(file_name).suffix)
+    file_name = str(file_name)
+    return Path(f"{file_name[:index]} - ManimCE v{__version__}{file_name[index:]}")
```

Comment:
Personally I'd prefer for spaces not to be in file names; it eases terminal use and I think ultimately looks better